### PR TITLE
Creator Showcase submission instructions

### PR DESCRIPTION
**Pull request by shannonbux**
@berraknil here it is.

What fields would someone fill in if they want to show their site showcase submissions as part of their "portfolio"?

### CODE REVIEW

File: docs/docs/submit-to-creator-showcase.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+title: Submit to Creator Showcase
+---
+
+Want to be a part of the [Creator Showcase](link TBA)? Follow these instructions.
+
+## Steps
+
+There are only two major steps :)
+
+1.  If this is your first contribution to the Gatsby open source repo, follow the [Contribution guidelines](https://next.gatsbyjs.org/docs/how-to-contribute/#contributing-to-the-repo) to get push access rights.
```
Comment (pieh) [inline docs/docs/submit-to-creator-showcase.md]: "to get push access rights" might be misleading here, as people would create pull request and not pushing to our repository.

I think common steps could be copied from https://github.com/gatsbyjs/gatsby/blob/f19287ec91505501cd1332345c4f0a414fabded6/docs/docs/site-showcase-submissions.md and change "edit `sites.yml`" step to edit `creators.yml` from below.

File: docs/docs/submit-to-creator-showcase.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  github: 'https://github.com/githubusername'
+  image: image.jpg
+  for_hire: true
+  hiring: false
+  portfolio: true
+```
+
+### Review process
+
+By default, all eits submitted to the Creator Showcase will be reviewed through the regular PR approval and merge process.
```
Comment (macklinu) [inline docs/docs/submit-to-creator-showcase.md]: s/eits/edits

**Comment by berraknil** [comment]
A note on images, they need to be uploaded to a specific directory `docs/community/images` as a jpg and should be a square with 500px minimum (ideally 720px) and 1000px maximum resolution. 

eg 500px x 500px or 1000px x 1000px

Also in case there are people who already sent their websites to Showcase but have not filled out the "built_by" field then (because it didn't exist until recently) it should be noted that they should go back and edit sites.yml as well to add themselves there to make sure their portfolio appears.

If you want I can edit the file directly on here but didn't want to add any changes before asking you first @shannonbux

**Comment by shannonbux** [comment]
Hi Berrak! Those edits sound great! Go ahead and make them if you'd like,
or I can get to them in a couple hours.

**Comment by berraknil** [comment]
@shannonbux added all the edits I could think of, let me know if you think there's something unclear/needs more explanation etc. Thanks!

**Comment by pieh** [comment]
What's the status of this PR? Should it be merged?

**Comment by berraknil** [comment]
I think everything is in place, @shannonbux is there anything missing/preventing from merging this? If it's good to go we can merge and then I will link it in the Become a Creator button on the page itself.

**Comment by calcsam** [comment]
Let's merge this in. If we need any follow-up we can put in subsequent PR.

**Pull request merged**